src/openforms/js/components/admin/form_design/AuthPluginAutoLoginField.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import Field from '../forms/Field';
+import {Checkbox} from '../forms/Inputs';
+
+const AuthPluginAutoLoginField = ({availableAuthPlugins, selectedAuthPlugins, selectedAuthPlugin, onChange, errors}) => {
+    let filteredPlugins = availableAuthPlugins.filter(plugin => selectedAuthPlugins.includes(plugin.id))
+    const authCheckboxes = filteredPlugins.map(plugin => {
+        const providedAttributes = (
+            <FormattedMessage
+                description="Auth plugin provided attributes suffix"
+                defaultMessage="(provides {attrs})"
+                values={{attrs: plugin.providesAuth.join(', ')}}
+            />
+        );
+        const label = (
+            <>
+                {plugin.label}
+                {plugin.providesAuth.length ? (<>{' '}{providedAttributes}</>) : null}
```

Comment:
Either we should refactor this duplication of code into a separate component, or not display it. I'm in favour of the last option, since the auth plugins with provided attributes are already displayed before.